tests/test_utils/test_transactiontestcase.py

```diff
@@ -1,7 +1,27 @@
+
```

Comment:
Can you have a look at code source ? I don't see any extra newline. I think the diff is a bit misleading.